kernel/byterun/coq_interp.c

```diff
@@ -712,26 +712,26 @@ value coq_interprete
           } else {
             /* The recursif argument is an accumulator */
             mlsize_t num_args, i;
+            value block;
```

Comment:
Warning this is dangerous, and need to be carefully checked.
The risk is that block can contains a value that will be modified by the GC.
So we need to be sure that block is declare as a root when potential call to the GC (Alloc_small)

Reply:
No need, this idiom is fine. Notice that `block` is always the target of `Alloc_small` (either that or it is no longer used). So, it does not need to be marked as a root.